build/compile/LzmaArchive.targets

```diff
@@ -115,11 +115,7 @@
       </GenerateNuGetPackagesArchiveVersion>
       <PropertyGroup>
         <NugetPackagesArchiveName>nuGetPackagesArchive.$(NuGetPackagesArchiveVersion).lzma</NugetPackagesArchiveName>
```

Comment:
Yea, looks like the math should stay in C#. Can we move that task to our pre-build, and generate the props there?